docs/developer/environment-setup.md

```diff
@@ -10,9 +10,9 @@ We're always happy to help new developers get started
 
 Gravity bridge has three major components
 
-[The Gravity bridge Solidity](https://github.com/Gravity-Bridge/Gravity-Bridge/tree/main/solidity) and associated tooling. This requires NodeJs
-[The Gravity bridge Cosmos Module and test chain](https://github.com/Gravity-Bridge/Gravity-Bridge/tree/main/module). this requires Go.
-[The Gravity bridge tools](https://github.com/Gravity-Bridge/Gravity-Bridge/tree/main/orchestrator) these require Rust.
+[The Gravity bridge Solidity](https://github.com/umee-network/Gravity-Bridge/tree/main/solidity) and associated tooling. This requires NodeJs
```

Comment:
I don't think we should update this URLs. Ideally these changes will go upstream.